The original configure did not find ImageMagick and thus did not build it.
This patch fixes this.
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/pstoedit-3.78.orig/configure.ac b/pstoedit-3.78/configure.ac
index e8d9e59..4d188ad 100644
--- a/pstoedit-3.78.orig/configure.ac
+++ b/pstoedit-3.78/configure.ac
@@ -313,7 +313,7 @@ if test "x${PSTOEDIT_GCC_OK}" = xno ; then
 else
       if test -n "$PKG_CONFIG"; then
         AC_MSG_CHECKING(ImageMagick flags through pkg-config)
-	PKG_CHECK_MODULES(LIBMAGICK, ImageMagick++, 
+	PKG_CHECK_MODULES(LIBMAGICK, ImageMagick, 
 		HAVE_LIBMAGICK=yes 
 		LIBMAGICK_CFLAGS="-DHAVE_MAGIC $LIBMAGICK_CFLAGS" 
 		LIBMAGICK_LDFLAGS="$LIBMAGICK_LIBS" , 
